presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -1036,13 +1157,19 @@ public SingleStreamSpiller create(List<Type> types, SpillContext spillContext, L
                 public ListenableFuture<?> spill(Iterator<Page> pageIterator)
                 {
                     checkState(writing, "writing already finished");
+                    if (failSpill) {
+                        return immediateFailedFuture(new PrestoException(GENERIC_INTERNAL_ERROR, "Spill failed"));
```

Comment:
I'm wondering if those cases add much value. They clearly increase the number of combinations we are testing by on order of magnitude...